tools/jdk/BUILD.java_tools

```diff
@@ -66,6 +66,18 @@ java_toolchain(
     ],
 )
 
+patch_11 = [
+    # In JDK9 we have seen a ~30% slow down in JavaBuilder performance
+    # when using G1 collector and having compact strings enabled.
+    "-XX:+UseParallelOldGC",
+    "-XX:-CompactStrings",
+    # override the javac in the JDK.
+    "--patch-module=java.compiler=$(location :java_compiler_jar)",
+    "--patch-module=jdk.compiler=$(location :jdk_compiler_jar)",
+]
+
+patch_14 = []
+
 # A toolchain that targets java JAVA_LANGUAGE_LEVEL.
 java_toolchain(
```

Comment:
Here are two dependencies on the old compiler: "javac" parameter and "tools" parameter. In case of Java 14 you can leave the out (the params are optional). 

 They old compiler is probably never used, but the dependency pulls down unnecessary artifacts.

I know I suggested to you before to use variables, however with three different parameters (after the change) it seems nicer to copy the whole target for Java14.